lib/Conversion/TritonGPUToLLVM/MemoryOpToLLVM.cpp

```diff
@@ -132,6 +132,84 @@ struct LocalStoreOpConversion
   const TargetInfoBase &targetInfo;
 };
 
+struct LocalLoadOpConversion : public ConvertOpToLLVMPattern<LocalLoadOp> {
```

Comment:
I assume this code is identical to the code that used to be in ConvertLayoutOpToLLVM.cpp?

(Since this PR is already very large, it might be good to split the cleanups into a separate PR.  What I'd do is squash your local history down to a single commit, and then git reset --soft HEAD^.  Then you can git commit -p to select the cleanups, and then git commit -a to commit everything else.)

Reply:
I've already squashed quite lot of local commits, so the commit for this specific change is missing, but I can revert it back manually.